java/src/main/java/com/logicalclocks/hsfs/engine/FeatureGroupEngine.java

```diff
@@ -33,13 +46,16 @@
 
 import java.io.IOException;
 import java.text.ParseException;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.TimeoutException;
 
 public class FeatureGroupEngine {
```

Comment:
Inconsisten with `FeatureViewEngine` it doesnt have a base class to extend from.

Reply:
Not sure what can I do here at this point. It seems to me that there will be v2 of this. I will gather all our comments in. design log. But and we can address all of this after we stabilize this PR